walkingastro.lic

```diff
@@ -14,16 +14,13 @@ class WalkingAstro
   end
 
   def get_telescope
-    fput('stop play')
-    case DRC.bput('get my telescope', 'You get a', 'You need a free hand to pick that up.', 'You are already holding that.', 'You should stop practicing your Athletics skill before you do that')
-    when 'You get a'
+    case DRCMM.get_telescope 
```

Comment:
💡 The way a few of the DRCMM helpers are used makes me wonder if they should be predicates. I won't have a strong opinion until I see more than one usage, though.